src/yamaoka/commands/admin/clear.command.ts

```diff
@@ -31,6 +32,20 @@ export default class ClearCommand extends BaseCommand<CommandType.SLASH_COMMAND>
 
     if ("bulkDelete" in channel) {
       await channel.bulkDelete(amount);
+      argument.reply({
+        embeds: [
+          JSON.parse(
+            JSON.stringify(embeds.Success).replace(
+              "%description%",
+              `${amount} ${
+                amount - 1 ? "messages" : "message"
+              } was deleted in <#${channel.id}>!`
+            )
+          ),
+        ],
+        ephemeral: true,
+      });
```

Comment:
The code is quite bad because readability is not that nice.
You can avoid using `.parse()` and `.stringify()` methods.
Also it's better to have logical parts as calculation and creation of description string separate.

You can do something like that

We use spread operation there to avoid changing initial embed.
For example:
You can do something like this `const success = embeds.Success`
Then you want to change description `success.description = "test"`
Okay, it's nice. You changed description, but what actually happening?
With that line of code you changing description of initial object because it's not "copying". You can think of that situation as `const success` becomes a link to object that you just assigned.

So as you can see we doing `spread` there.
`const successEmbed = { ...embeds.Success }`
`const plurarizeWord = amount - 1 ? "messages" : "message"`
`const description = description implementation here`
`successEmbed.description.replace("%description%", description)`
`argument.reply({ embeds: [successEmbed], ephemeral: true })`

Reply:
"Also it's better to have logical parts as calculation and creation of description string separate."

Yea I thought about that, I could make a separate embed (like in your example) or work with one in a JSON. This is why then I uploaded merge wasn't sure about syntax.